resources/eventing/charts/controller/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: controller
+description: A Helm chart for Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+appVersion: 1.16.0
```

Comment:
I think we are not using appVersion in Kyma yet. Do you plan to use it in this chart in the future? If yes I guess 1.16.0 is not the version of the controller. I think it makes sense to have it if we have proper versioning for the controller.

Reply:
Removed `appVersion` for now